agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -4458,6 +4464,49 @@ bool backhaul_manager::handle_slave_channel_selection_response(ieee1905_1::CmduM
     return send_cmdu_to_bus(cmdu_tx, controller_bridge_mac, bridge_info.mac);
 }
 
+bool backhaul_manager::handle_backhaul_steering_request(ieee1905_1::CmduMessageRx &cmdu_rx,
+                                                        const std::string &src_mac)
+{
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received BACKHAUL_STA_STEERING message, mid=" << std::hex << int(mid);
+
+    auto bh_sta_steering_req = cmdu_rx.getClass<wfa_map::tlvBackhaulSteeringRequest>();
+    if (!bh_sta_steering_req) {
+        LOG(ERROR) << "Failed cmdu_rx.getClass<wfa_map::tlvBackhaulSteeringRequest>(), mid="
```

Comment:
```suggestion
        LOG(WARNING) << "Failed cmdu_rx.getClass<wfa_map::tlvBackhaulSteeringRequest>(), mid="
```
Since this message comes from outside, it is not an "error" (in the sense that something is wrong with our system). Rather , it is invalid input.

Doesn't matter much of course, but we should probably do this right.

Reply:
accepted!